README.md

```diff
@@ -228,6 +228,29 @@ After dealing with *CUDA* installing just install *Optix SKD*.
 
 - **[OptiX SDK](https://developer.nvidia.com/designworks/optix/download)** 
 
+### Android Build
+
+**Required:**
+
+- **[Android Studio](https://developer.android.com/studio)**
+- **[NDK r22b](https://developer.android.com/ndk/downloads#stable-downloads)**
+- **[Java 8](https://www.java.com/download/)**
+- **[Ninja](https://ninja-build.org/)**
```

Comment:
I think its nice, but maybe not required?

Reply:
what do you mean? Do you want me to remove the Ninja limit generator?

Reply:
oh, it doesnt build with make?

Reply:
nah, we wanted only Ninja build on Android (but we can change it?)

Reply:
i think you can build with make, its just slooow

Reply:
okay, I will add it to validation

Reply:
done